README.md

```diff
@@ -31,5 +33,5 @@ import { TeliStandardLayout } from '@telicent-oss/ds';
 ```sh
 git clone https://github.com/Telicent-oss/telicent-ds.git
 cd telicent-ds
-yarn storybook
-```
\ No newline at end of file
+yarn && yarn storybook
```

Comment:
I'm not _too_ worried about the stories ATM. 
But could you provide a screenshot so I can create a ticket?

**Site-note**: We'll need to write some (very simple) guidelines for the stories.

Reply:
Here's a couple of screenshots:
1. The page exactly as I see it (note the text is very faint)
2. The text within the page highlighted see it can be seen.

If it's any help I'm running macOS 14.4 with the Appearance set to 'Light'.

Let me know if you need any more information to support the ticket. 
![Screenshot 2024-03-18 at 14 17 57](https://github.com/telicent-oss/telicent-ds/assets/148454215/471bca5c-0044-459a-a9a9-12d89c992edd)  

![Screenshot 2024-03-18 at 14 18 13](https://github.com/telicent-oss/telicent-ds/assets/148454215/87d8be14-7e65-4c9c-9b6b-90df5d3f6578)

Reply:
No worries!  I think I'll keep the 'light' theme on ... it seems to shake out some interesting quirks of the user interface in various parts of CORE.